src/modules/personenkontext/persistence/personenkontext.entity.ts

```diff
@@ -8,7 +8,13 @@ import { PersonEntity } from '../../person/persistence/person.entity.js';
 @Unique({ properties: ['personId', 'organisationId', 'rolleId'] })
 export class PersonenkontextEntity extends TimestampedEntity {
     @AutoMap()
-    @ManyToOne({ columnType: 'uuid', cascade: [], ref: true, nullable: false, entity: () => PersonEntity })
+    @ManyToOne({
+        columnType: 'uuid',
+        cascade: [Cascade.REMOVE],
```

Comment:
It's necessary otherwise cascading doesn't work. (tried it in 1 direction). I think it's because it's a bidirecitonal relationship?

Also if we delete the Personenkontext the person isn't deleted since the person is the parent.